src/ObjCRuntime/Runtime.cs

```diff
@@ -206,6 +206,7 @@ internal unsafe struct InitializationOptions {
 		internal static unsafe InitializationOptions* options;
 
 #if NET
+		[Preserve (AllMembers = true)]
```

Comment:
Could you maybe just add this class + method dynamically in the rewriter? That way it would only be present when needed.

Reply:
If the method is called from somewhere, then you won't need the Preserve attribute.